### PR TITLE
ConselJS 5 migration

### PR DESCRIPTION
**Pull request by anonymoussprocket**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+
 import Root from './containers/Root';
 import { configureStore, history } from './store/configureStore';
 import './utils/i18n';
 import './styles/app.global.scss';
 
 const store = configureStore();
 
+const logger = log.getLogger('conseiljs');
+logger.setLevel('debug', false);
```
Comment (vishakh) [inline src/index.tsx:16]: In an ideal world, this would be a configuration item.

File: src/utils/wallet.ts [hunk 2]
```diff
@@ -80,25 +60,94 @@ export async function loadPersistedState(walletPath: string, password: string):
...
-    return knownTokenContracts.filter(token => token.network === network);
+/**
+ * Saves a wallet to a given file.
+ *
+ * @param {string} filename Path to file
+ * @param {Wallet} wallet Wallet object
+ * @param {string} passphrase User-supplied passphrase
+ * @returns {Promise<Wallet>} Wallet object loaded from disk
+ */
+export async function saveWallet(filename: string, wallet: Wallet, passphrase: string): Promise<Wallet> {
```
Comment (vishakh) [inline src/utils/wallet.ts:90]: Should these wallet functions not be in ConseilJS? That's where we always had them. Why are we moving them?

Comment (anonymoussprocket) [reply src/utils/wallet.ts:90]: No, conseiljs is a blockchain data interface library for talking to nodes and indexers. "file wallet"s are clearly a wallet app feature.

File: src/index.tsx (hunk 1 above)
Comment (anonymoussprocket) [reply src/index.tsx:16]: fixed

**Pull request merged**